src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralParsingTests.cs

```diff
@@ -0,0 +1,1420 @@
+    }
+
+    [Fact]
+    public void AtThenDollarThenAt3()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = @@$@"" "";
+    }
+}";
+
+        CreateCompilation(text).VerifyDiagnostics(
+            // (6,17): error CS9125: Cannot mix verbatim and raw strings
+            //         var v = @@$@" ";
+            Diagnostic(ErrorCode.ERR_CannotMixVerbatimAndRawStrings, @"@@$@""").WithLocation(6, 17));
+    }
+
+    [Fact]
+    public void AtThenDollarThenAt4()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = @@$@"""""" """""";
+    }
+}";
+
+        CreateCompilation(text).VerifyDiagnostics(
+            // (6,17): error CS9125: Cannot mix verbatim and raw strings
+            //         var v = @@$@""" """;
+            Diagnostic(ErrorCode.ERR_CannotMixVerbatimAndRawStrings, @"@@$@""""""").WithLocation(6, 17));
+    }
+
+    [Fact]
+    public void AtThenDollarThenAt5()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = @@$$@;
+    }
+}";
+
+        CreateCompilation(text).VerifyDiagnostics(
+            // (6,17): error CS9125: Cannot mix verbatim and raw strings
+            //         var v = @@$$@;
+            Diagnostic(ErrorCode.ERR_CannotMixVerbatimAndRawStrings, "@@$$@").WithLocation(6, 17));
+    }
+
+    [Fact]
+    public void AtThenDollarThenAt6()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = @@$$@"";
+    }
+}";
+
+        CreateCompilation(text).VerifyDiagnostics(
+            // (6,17): error CS9125: Cannot mix verbatim and raw strings
+            //         var v = @@$$@";
+            Diagnostic(ErrorCode.ERR_CannotMixVerbatimAndRawStrings, @"@@$$@""").WithLocation(6, 17),
+            // (6,24): error CS1002: ; expected
+            //         var v = @@$$@";
+            Diagnostic(ErrorCode.ERR_SemicolonExpected, "").WithLocation(6, 24));
+    }
+
+    [Fact]
+    public void AtThenDollarThenAt7()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = @@$$@"" "";
+    }
+}";
+
+        CreateCompilation(text).VerifyDiagnostics(
+            // (6,17): error CS9125: Cannot mix verbatim and raw strings
+            //         var v = @@$$@" ";
+            Diagnostic(ErrorCode.ERR_CannotMixVerbatimAndRawStrings, @"@@$$@""").WithLocation(6, 17));
+    }
+
+    [Fact]
+    public void AtThenDollarThenAt8()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = @@$$@"""""" """""";
+    }
+}";
+
+        CreateCompilation(text).VerifyDiagnostics(
+            // (6,17): error CS9125: Cannot mix verbatim and raw strings
+            //         var v = @@$$@""" """;
+            Diagnostic(ErrorCode.ERR_CannotMixVerbatimAndRawStrings, @"@@$$@""""""").WithLocation(6, 17));
+    }
+
+    [Fact]
+    public void DollarsWithoutQuotes1()
+    {
+        var text = @"
+class C
+{
+    void M()
+    {
+        var v = $$;
```

Comment:
Could we also test a lone dollar sign `var v = $;`?

Reply:
Done.